guides/common/modules/proc_synchronizing-the-system-clock-with-chronyd.adoc

```diff
@@ -4,17 +4,17 @@
 To minimize the effects of time drift, you must synchronize the system clock on the base operating system on which you want to install {ProductName} with Network Time Protocol (NTP) servers.
 If the base operating system clock is configured incorrectly, certificate verification might fail.
 
+ifdef::satellite[]
```

Comment:
I'd also keep this in `foreman-el` and `katello` because this is documenting a system tool.

Reply:
Applied.